include/lsqecc/layout/ascii_layout_spec.hpp

```diff
@@ -120,6 +122,7 @@ class LayoutFromSpec : public Layout{
     const std::vector<SurfaceCodeTimestep>& distillation_times() const override {return cached_distillation_times_;};
     const std::vector<Cell>& ancilla_location() const override {return cached_ancilla_locations_;}
     const std::vector<Cell>& dead_location() const override {return cached_dead_cells_;}
+    const std::vector<Cell>& y_states() const override {return cached_y_states_;}
```

Comment:
Might be worth changing this to `predistilled_y_states` to distinguish it from the `ancilla_location` (which probably deserves a better name too :sweat_smile: